www/docs/en/dev/guide/overview/project-structure.md

```diff
@@ -0,0 +1,100 @@
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+title: Project Structure of a Cordova Project
+toc_title: Project Structure
+description:
+---
+
+# Project Structure
+
+## CLI's Default Directory Structure
+
+A project created with Cordova CLI comes with the follow directory structure by default:
+
+```text
+myapp/
+├── config.xml
+├── node_modules/
+├── package.json
+├── platforms/
+└── plugins/
+└── www
+```
+
+### `package.json`
+
+TODO
+
+### `config.xml`
+
+Configures your application and allows you to customize the behavior of your project. See also [config.xml reference documentation][config.xml ref]
+
+### `www/`
+
+Contains the project's web artifacts, such as .html, .css and .js files. As a cordova application developer, most of your code and assets will go here. They will be copied on a `cordova prepare` to each platform's www directory. The www source directory is reproduced within each platform's subdirectory, appearing for example in `platforms/ios/www` or `platforms/android/assets/www`. Because the CLI constantly copies over files from the source www folder, you should only edit these files and not the ones located under the platforms subdirectories. If you use version control software, you should add this source www folder, along with the merges folder, to your version control system.
+
+### `node_modules/`
+
+This directory contains all of the checked-out packages from the npmjs registry.
+
+When adding a Cordova platform or plugin with the `cordova platform add` and `cordova plugin add` command, these platforms and plugins are fetched from the npmjs registry and downloaded into the `node_modules/` directory.
+
+Cordova will then copy the necessary Cordova platform and plugin  source code from the `node_modules` directory and place them into the appropriate location for Cordova to function.
+
+It also contain scripts that is used during the `cordova prepare` and `cordova build` for each platform.
+
+The `node_modules` directory is  the original unedited source of truth and nothing should be edited in this directory. Additionally, this directory should not be checked into any version control system.
+
+For more details, see [npmjs folders documentation](https://docs.npmjs.com/cli/v7/configuring-npm/folders#node-modules).
+
+### `platforms/`
+
+Contains all the source code and build scripts for the platforms that you add to your project.
+
+> **WARNING:** When using the CLI to build your application, you should not edit any files in the /platforms/ directory unless you know what you are doing, or if documentation specifies otherwise. The files in this directory are routinely overwritten when preparing applications for building, or when plugins are re-installed.
+
+### `plugins/`
+
+Any added plugins will be extracted or copied into this directory.
+
```

Comment:
Do we want the same warning here about not checking in the `/plugins` folder to version control?

Reply:
Yeah, I think a warning should be added to not commit the plugins directory to version control.